### DISCUSSION TITLE
Please share your ideas about this project

### DISCUSSION THREAD

**Discussion by Adela-aman** [SoHailaNekHat/Project-of-SE#2] [Ideas]
How is multiplication table in Java? 
Everyone will push one number multiplication table;

**Comment by KhatimaSadat** [comment]
It is good idea but I suggest one simple calculator 
What about it ?

**Reply by Adela-aman** [reply]
That's all right

**Reply by KhatimaSadat** [reply]
Ok what about other girls ?

**Reply by KhatimaSadat** [reply]
They are agree ?
